Frameworks/Foundation/NSDecimal.mm

```diff
@@ -0,0 +1,592 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#import <Foundation/NSDecimal.h>
+#import <StubReturn.h>
+#include <algorithm>
+
+#define EXPONENT_MAX 127
+#define EXPONENT_MIN -128
```

Comment:
this is internal, and I don't need to be presented as a var.
preprocessor placement is fine for it's limited use.

---
In reply to: [75560569](https://github.com/Microsoft/WinObjC/pull/896#discussion_r75560569) [](ancestors = 75560569)